Libraries/MiscDrivers/Display/tft_ssd2119.c

```diff
@@ -926,10 +927,10 @@ int MXC_TFT_Init(void)
     memset(&images_header, 0, sizeof(Header_images_t));
 
     // Is there any image data to work with?
-    if (_bin_start_ != _bin_end_) {
+    if ((unsigned int *)&_bin_start_ != (unsigned int *)&_bin_end_) {
```

Comment:
Is this cast necessary, or should this reference the first field (unsigned int offset2info_palatte) of Header_images_t instead?

Reply:
Casting is for readability purposes only. We're checking if any data inserted in this specific block.

Reply:
These casts are the opposite of "readable" and like any cast in C, carry the risk of mis-aligned accesses and access to unallocated memories.